tests/kolainst/destructive/staged-deploy.sh

```diff
@@ -98,6 +98,11 @@ EOF
     syncfs=$(journalctl -b -1 -u ostree-finalize-staged --grep='Completed syncfs.*for system repo' | wc -l)
     assert_streq "$syncfs" 2
 
+    # And verify there were no failures to unmount /etc or /sysroot
```

Comment:
When doing soft-reboot I think we keep the full journal, but on normal reboot this would only catch failures before /var umount.
I think we want to duplicate this test for the soft-reboot case.

Best would be to have the VM serial output, but I don't know what kola can do.

Reply:
Yeah you're right, moved to soft reboot.

> Best would be to have the VM serial output, but I don't know what kola can do.

We do gather it, but it's currently outside what external tests can gate on with coreos-assembler. I'm working in the background on some things like https://github.com/bootc-dev/podman-bootc/issues/99 which can help form the core of a more powerful container-native testing framework.